go/base/management/commands/go_setup_env.py

```diff
@@ -582,7 +582,13 @@ def create_billing_api_supervisord_conf(self):
             fp.write(self.auto_gen_warning)
             cp = ConfigParser()
             cp.add_section(section)
-            cp.set(section, "command", "./go-admin.sh runbillingserver")
+            cp.set(
+                section, "command",
+                "twistd -n web --class=go.billing.api.billing_api_resource "
+                "--port=910%(process_num)s")
+            cp.set(
+                section, "environment",
+                "DJANGO_SETTINGS_MODULE='go.settings',PYTHONPATH='.'")
```

Comment:
Yes, according to the docs it's correct: http://supervisord.org/configuration.html#supervisord-section-example